README.md

```diff
@@ -0,0 +1,13 @@
+# WMSE Scripts
+This repository contains miscellaneous scripts used by [Wikimedia Sverige (WMSE)](https://www.wikimedia.se). Most of these scripts are single file. 
+
+* fortnox_huvudbok_processing - *???*
+* python_scripts
+  * extension_documentation.py - *generates wikitext strings to use on extension pages on the [MediaWiki-wiki](https://www.mediawiki.org).*
+  * wikimania_program_list.py - *generate wikitext for program list used for Wikimania 2019.*
+* drive_calc_getSheetName.js - *???*
```

Comment:
Google script for the _Burnchart_ Calc document. Used to copy tab name to a cell and to calculate personnel costs.

Reply:
Sorry that is a separate script where this is included.
"Google script for Calc to create a function which returns the name of the sheet where a given cell resides."